src/pages/Admin/Details/Details.scss

```diff
@@ -0,0 +1,23 @@
+@import "scss/constants";
```

Comment:
A replacement for the old `Details.styles.ts`  with the capability of now easily reusing `scss/constants.scss`. The need for a bit of style/layout adjustment came from the `ButtonNG` being different component than `Button` and the replacement of styled components with the projects own BEM style SCSS files.